test/bdd/go.mod

```diff
@@ -13,7 +13,7 @@ require (
 	github.com/cucumber/godog v0.8.1
 	github.com/fsouza/go-dockerclient v1.6.0
 	github.com/google/uuid v1.1.2
-	github.com/hyperledger/aries-framework-go v0.1.5-0.20201204175718-47143e0eada8
+	github.com/hyperledger/aries-framework-go v0.1.5
 	github.com/trustbloc/edge-core v0.1.5-0.20201126210935-53388acb41fc
 	github.com/trustbloc/edge-service v0.0.0
 	github.com/trustbloc/trustbloc-did-method v0.1.5-0.20201203214019-c56f43ad3f6e
```

Comment:
did-method to release?

Reply:
@sandrask This is WIP. I'll update once did-method is released.

Reply:
update did-method. now waiting for edv.